front/src/pages/people/People.tsx

```diff
@@ -53,6 +55,14 @@ export function People() {
 
   const theme = useTheme();
 
+  const setContextMenu = useOpenContextMenu();
+  const setActionBar = useOpenActionBar();
+
+  useEffect(() => {
```

Comment:
We avoid to put useEffects in Components that have children at all cost as this way of cofing can trigger unexpected rerenders.

We have two solutions to avoid the issue:
- create a component that will be beside the Component with Children. This is what we do with GenericEntityTableData for example. This component returns <></>, this way the impact of this component re-rendering is low
- use the only exception allowed in AuthAutoRouter. If you think about it, the only reason moment we really need a useEffect is when the page is loading (so very high on the hierarchy). Then we should be able to use imperative calls to trigger app logic.

Here maybe you can re-use GenericEntityTableData